.github/workflows/source.yml

```diff
@@ -30,7 +30,12 @@ jobs:
           vcs-repo-file-url: |
             https://raw.githubusercontent.com/ros2/ros2/galactic/ros2.repos
             https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/ros2_control/ros2_control.repos
-          colcon-mixin-name: coverage-gcc
+          colcon-mixin-name: |
```

Comment:
Shouldn't this be colcon-defaults:

Reply:
Thank you for catching that! Fixed in e7c1844.